### DISCUSSION TITLE
Lost setting

### DISCUSSION THREAD

**Discussion by GrimreeprSask** [Open-Shell/Open-Shell-Menu#1748] [General]
I had an issue with windows update that bricked my whole system, I had some settings in openshell that I need to recover into the new install.  I still have all the windows files in the windows.old install, but I can't remember the file names of some of the textures I was using for the taskbar and background.  I am sure I still have them, but how do I go about extracting/viewing the settings from the old install so I can find those file names?

**Comment by ge0rdi** [comment]
`Open-Shell` stores its settings in registry.
So if you can access the old registry hive you can load it in `Regedit` and then look at the `SOFTWARE\OpenShell` subkey.

**Comment by GrimreeprSask** [comment]
Awesome, thanks for the reply

**Reply by GrimreeprSask** [reply]
No help, that subkey just has the version setting and the path, nothing about settings at all

**Reply by ge0rdi** [reply]
You need to load user's registry hive.
These settings are per-user.

**Comment by GrimreeprSask** [comment]
Found it, it's in the user settings, the way it was written made it look like it was in software hive.

**Comment by GrimreeprSask** [comment]
/em does a happy dance.  All settings and textures are back where they are supposed to be.  Thanks again!
